.travis.yml

```diff
@@ -30,7 +30,7 @@ matrix:
       node_js:
         - 'lts/*'
     - name: "Develop UI Tests"
-      env: SRC_PATH='examples/gatsbygram' INSTALL_COMMAND='install' TEST_COMMAND='test'
+      env: SRC_PATH='examples/gatsbygram' INSTALL_COMMAND='run bootstrap' TEST_COMMAND='test'
```

Comment:
This will bootstrap (once) all gatsby packages, which is what we want here, correct?